<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>YouTube Video Downloader</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>391</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grid_layout">
     <item row="4" column="0">
      <widget class="QPushButton" name="button_download">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>161</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Download...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" rowspan="2">
      <widget class="QTreeView" name="tree_view"/>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="button_find">
       <property name="text">
        <string>Find Available Streams</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="Line" name="separator">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QGridLayout" name="grid_layout_inner">
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="group_box">
         <property name="minimumSize">
          <size>
           <width>161</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>161</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Stream Filters</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>16</y>
            <width>141</width>
            <height>81</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="vertical_layout">
           <item>
            <widget class="QCheckBox" name="check_box_progressive">
             <property name="text">
              <string>Video (Progressive)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="check_box_adaptive">
             <property name="text">
              <string>Video (Adaptive)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="check_box_audio">
             <property name="text">
              <string>Audio Only</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_image">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>161</width>
           <height>90</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>161</width>
           <height>90</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_edit_url">
       <property name="placeholderText">
        <string>YouTube Video URL</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_title">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_bar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>313</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="status_bar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
